submissions/unabyband/oop_exercise/index.js

```diff
@@ -0,0 +1,122 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/unabyband/a-tiny-JS-world
+   Web app: https://unabyband.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+  constructor(species, name, gender, legs, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  printInhabitant() {
```

Comment:
(1) Separation of concerns (remember requirements re Frogger?)
This method forms an output and also conducts output.

(2) You may construct arrays from any legal JS expressions. Consider an example below
```
[
  this.propName,
  `prefix ${this.PropName} suffix`,
  arrayVariable.join("glue "),
  condition ? expressionIfTrue : "stringIfFalse",
]

```